components/sbm-recipes-boot-upgrade/pom.xml

```diff
@@ -102,4 +110,4 @@
             </snapshots>
```

Comment:
Could you also remove this repository information? I know it didn't came in with your commit but I can't see why it's required here and it just came to my eye. If we need it (not sure why we would) it should go into parent pom. Thanks :)